src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -307,15 +300,6 @@ def query_top_tags(
     :return: Returns the row with the value, the aggregate and the count if the query was successful
              Returns None if query was not successful which causes the endpoint to return early
     """
-    with sentry_sdk.start_span(
-        op="discover.discover", description="facets.filter_transform"
-    ) as span:
```

Comment:
should `start_span` be removed here completely?

Reply:
Yep, the filter transforms that happened here now happen within the QueryBuilder, which has its own spans

Reply:
ah, got it 👍